BackEnd/src/main/java/com/exadel/carpoolfree/model/Drive.java

```diff
@@ -31,11 +20,11 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(name = "time_start")
-    private LocalDateTime timeStart;
+    @Column(name = "start_time")
+    private String startTime;

```

Comment:
LocalDateTime was actually correct, let's have it back :)

Reply:
fixed